src/platforms/hosted/probe_info.c

```diff
@@ -63,14 +69,15 @@ probe_info_s *probe_info_add(probe_info_s *const list, const bmp_type_t type, co
 	}
 
 	probe_info->type = type;
+	probe_info->vid  = vid ;
+	probe_info->pid = pid ;
 	probe_info->manufacturer = mfr;
 	probe_info->product = product;
 	probe_info->serial = serial;
 	probe_info->version = version;
 
 	probe_info->next = list;
-	return probe_info;
-}
+	return probe_info;}
```

Comment:
The `}` here should be on its own line, and the VID and PID assignments above introduce a whitespace error before the `;` on both lines, please run clang-format to fix these.

Reply:
Hmm, I thought I ran clang-format ... I notice that in my VSCode I have to explicitly save the file, rather than making an edit and just running my build task to get Clang-format to run.

Commit pushed.